lib/rage/cli.rb

```diff
@@ -4,6 +4,23 @@
 require "rack"
 require "rage/version"
 
+# mock fiber methods as RSpec tests don't run concurrently
+class Fiber
+  def self.schedule(&block)
+    fiber = Fiber.new(blocking: true) do
+      Fiber.current.__set_id
+      Fiber.current.__set_result(block.call)
+    end
+    fiber.resume
+
+    fiber
+  end
+
+  def self.await(fibers)
+    Array(fibers).map(&:__get_result)
+  end
+end
+
```

Comment:
Unfortunately, this is the same as requiring the file 😞 
This patch also applies to the server process, effectively making all code inside the application synchronous.

You can verify this by creating a new Rage app and adding an action that calls `sleep 1`. Sending however many requests in parallel to this action should only take one second, but with these changes Rage will be processing the requests one by one.

Instead, we only want these changes applied to the [console](https://github.com/rage-rb/rage/blob/master/lib/rage/cli.rb#L147-L156) method.